auto_py_to_exe/web/js/i18n.js

```diff
@@ -90,6 +98,7 @@ const translationMap = {
                 pt_br: 'Saída',
                 sp: 'Salida',
                 tr: 'Çıktı',
+                th:'Output',

```

Comment:
If the translation is not different to `en`, you can leave it out as `en` will be defaulted to.